infrastructure/main.tf

```diff
@@ -87,7 +41,7 @@ module "db-rd-caseworker-ref-v16" {
   admin_user_object_id = var.jenkins_AAD_objectId
   business_area        = "cft"
   common_tags          = var.common_tags
-  component            = var.component-v16
+  component            = var.component
```

Comment:
if we change this, new DB is going to be created?

Reply:
reverted